app/controllers/api/status_controller.rb

```diff
@@ -13,7 +13,7 @@ def check
     end
     render json: @projects.to_json({
       only: Project::API_FIELDS,
-      methods: [:package_manager_url, :stars, :forks, :keywords, :latest_stable_release],
+      methods: [:package_manager_url, :stars, :forks, :keywords, :latest_stable_release, :latest_download_url],
```

Comment:
So my understanding of to_json is that latest_download_url has to be in methods rather than in `only` (where it already was) because it's a method instead of a field. I'm not 100% confident I have that right.